src/main/webapp/index.jsp

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="utf-8">
+<title>canvas_courses_report</title>
```

Comment:
Would give it a better title. It will go on the browser bar and bookmarks so it should be readable.